cfme/tests/configure/test_access_control.py

```diff
@@ -410,19 +410,25 @@ def test_group_crud_with_tag(provider, tag_value, group_collection):
         * Save group
     """
     tag_for_create, tag_for_update = tag_value
+
+    if provider.key == 'vsphere55':
+        path = 'VM_Template-Folder'
+    else:
+        path = 'Discovered virtual machine'
+
```

Comment:
**Optional** The more pythonic way to write this statement would be :
```path = 'VM_Template-Folder' if provider.key == 'vsphere55' else 'Discovered virtual machine'```

Reply:
Thanks!